bears/xml/XMLBear.py

```diff
@@ -1,5 +1,26 @@
+import os
+import requests
+
 from coalib.bearlib.abstractions.Lint import Lint
 from coalib.bears.LocalBear import LocalBear
+from coalib.misc.Shell import escape_path_argument
+from coalib.settings.Setting import path
+
+
+def path_or_url(type1, type2):
+    '''
+    Checks if the argument is a ``url`` or a ``path``.
+
+    :param type1:       path
+    :param type2:       str
+    :return:            path or str
+    '''
+    def _path_or_url(dtdvalid):
+        if dtdvalid is not None and os.path.exists(dtdvalid):
+            return type1
```

Comment:
@AbdealiJK How to print it. A simple print statement isn't displaying anything in the error log of py.test